stories/Port.stories.tsx

```diff
@@ -104,6 +104,60 @@ export const Simple = () => (
   </div>
 );
 
+export const LinkingPortToPort = () => (
+  <div style={{ position: 'absolute', top: 0, bottom: 0, left: 0, right: 0 }}>
+    <Canvas
+      nodes={[
+        {
+          id: '1',
+          text: 'Node 1',
+          ports: [
+            {
+              id: '1-from',
+              width: 10,
+              height: 10,
+              side: 'SOUTH'
+            },
+            {
+              id: '1-to',
+              width: 10,
+              height: 10,
+              side: 'NORTH'
+            }
+          ]
+        },
+        {
+          id: '2',
+          text: 'Node 2',
+          ports: [
+            {
+              id: '2-from',
+              width: 10,
+              height: 10,
+              side: 'SOUTH'
+            },
+            {
+              id: '2-to',
+              width: 10,
+              height: 10,
+              side: 'NORTH'
+            }
+          ]
+        }
+      ]}
+      edges={[]}
+      node={<Node dragType="port" />}
+      onNodeLinkCheck={(_e, _fromNode, _toNode, _fromPort, toPort) => {
+        // Only allow port to port dragging
+        return toPort !== undefined;
```

Comment:
It would be nice if setting the Node's dragType to "port" it would also prevent dragging to a node, thoughts?
Currently it looks that `dragType` is only affecting from where you can start dragging and not where you may drag to.

Reply:
I'm open to that - want to update the PR to fix that too?

Reply:
That would be a breaking change tho, maybe merge this first and tackle that in a separate PR?